README.md

```diff
@@ -181,6 +181,9 @@ for more details on the `CHECK TABLE` statement.
 	[--<field>=<value>]
 		Extra arguments to pass to mysqlcheck. [Refer to mysqlcheck docs](https://dev.mysql.com/doc/en/mysqlcheck.html).
 
+	[--defaults]
+		Removes the "--no-defaults" flag normally passed to MySQL allowing it to use the default my.cnf or one specified with the MYSQL_HOME environment variable.
```

Comment:
The help text should focus less on the current implementation, and not use this "double negation" in the sense of "remove => no".

How about something like this:
```
Loads the environment's MySQL option files. Default behavior is to skip loading them to avoid failures due to misconfiguration.
```

Reply:
Fixed